development-defaults.ini

```diff
@@ -1,5 +1,5 @@
 
-path = "/uber"
+path = "/rams"
```

Comment:
Huh, GitHub gives me a 404 when I try to submit a PR. @md1024 can you make this change?